cdk/lib/cdk-stack.ts

```diff
@@ -51,6 +54,7 @@ export class CdkStack extends GuStack {
       },
       schedule: Schedule.rate(lambdaFrequency),
       monitoringConfiguration: {
+        alarmName: `High error % from ${app}-${this.stage}`,
```

Comment:
This resource is being replaced. AWS attempts to create a new alarm before deleting the old one, so we need to use a different name or this operation fails (you cannot have 2 alarms with the same name). As it happens the new name is a bit better anyway 😄